src/types/UiaTextRangeBase.cpp

```diff
@@ -409,6 +366,7 @@ IFACEMETHODIMP UiaTextRangeBase::GetAttributeValue(_In_ TEXTATTRIBUTEID textAttr
     return S_OK;
 }
 
+// TODO CARLOS: Completely rewrite this. this will be a bit of a pain :/

```

Comment:
TODO when? If not in this PR, leave the GH item ID instead of 'CARLOS'

Reply:
This actually got rewritten. Just didn't look forward to it.